docs/credentials-handling.md

```
@@ -115,14 +118,185 @@ spec:
 ```
 5. Continue following the OSC installation instructions while making sure your created RG is always the one used (the one in use also above), in peer-pods-cm and in configuring the default worker subnet
 
-#### Credentials Cleanup
+##### Credentials Cleanup
 1. Delete the identity when you delete OSC:
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)